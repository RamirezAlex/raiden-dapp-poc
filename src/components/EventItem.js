import React from 'react'
const EventItem = (props) => {
  const { event, count } = props
  return (
    <tr className={event.event}>
      <td>{ count }</td>
      <td>{ event.id }</td>
      <td>{ event.channelIdentifier }</td>
      <td>{ event.event }</td>
    </tr>
  )
}

export default EventItem
