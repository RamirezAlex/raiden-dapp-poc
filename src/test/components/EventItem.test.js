import React from 'react'
import EventItem from '../../components/EventItem'
import { create } from 'react-test-renderer'

describe('EventItem', () => {
  it('Shoulde render a EventItem', () => {
    expect(EventItem).not.toBeNull()
    const event = {
      id: 'log_0xe37b602261c301b249f0be08ab7da077db0d94372ea034278b57eab2ebecda59',
      channelIdentifier: 26,
      event: 'ChannelSettled'
    }

    const component = create(<EventItem event={event} count={4} />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
