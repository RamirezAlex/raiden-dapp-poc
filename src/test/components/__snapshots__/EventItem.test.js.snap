// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventItem Shoulde render a EventItem 1`] = `
<tr
  className="ChannelSettled"
>
  <td>
    4
  </td>
  <td>
    log_0xe37b602261c301b249f0be08ab7da077db0d94372ea034278b57eab2ebecda59
  </td>
  <td>
    26
  </td>
  <td>
    ChannelSettled
  </td>
</tr>
`;
